canonicaljson.py

```diff
@@ -126,17 +125,16 @@ def _unascii(s):
                 # leave as a \uNNNN escape
                 chunks.append(s[pos:end])
             else:
-                if PY3:   # pragma nocover
```

Comment:
there are some comments above this about python 2 behaviour which could probably do with cleaning up.

Reply:
Good call. I removed them all except the one about importing simplejson at the top. I think it is still valid for why that code exists. I can update it in a follow-up PR saying we use simplejson for backwards compatibility.